### DISCUSSION TITLE
portal2运行问题

### DISCUSSION THREAD

**Discussion by SuperSummer07** [Gamuxorg/bbs#327] [游戏或网站运行反馈]
游戏名称：

操作系统：20.04.1-Ubuntu

bug描述：无法启动

终端运行输出 ：./portal2_linux: error while loading shared libraries: libtcmalloc_minimal.so.4: cannot open shared object file: No such file or directory

自行尝试解决bug的方式和结果：按网站所说删掉/bin/libstdc++.so.6仍报此错误

**Comment by hyh19962008** [comment]
删/bin/libstdc++.so.6是解决libGL的错误，我再看看

**Comment by SuperSummer07** [comment]
下了个steam,steam自动把缺的库都装好了....完美运行
